vscode/test/ui-test/common/abstractServerTest.ts

```diff
@@ -0,0 +1,31 @@
+import { By, VSBrowser } from "vscode-extension-tester";
+
+/**
+ * @author Oleksii Korniienko <olkornii@redhat.com>
+ */
+
+export async function getServerConnector(connectorName,section) {
```

Comment:
I do not see much point why is this script called abstract. You are not extending it and it is not a class after all.

Reply:
ServerUtility